### ISSUE TITLE
Is this project still being supported?

### ISSUE THREAD

**Issue by ccpony** [andrewdavey/postal#162]

**Comment by tiesont** [comment]
Depends on what you mean. There are active forks (such as https://github.com/hermanho/postal). Are there features you're after that aren't implemented?

**Comment by jenniferhopkin12** [comment]
Reaching out to the project's maintainers or [garage door](https://garagedoorrepairshortpump.com/common-garage-door-problems-and-their-solutions/) directly might provide clarity. They're usually the best source of information regarding ongoing support.

**Comment by jamesbond867** [comment]
Yes, it appears the project is still actively supported. You might want to check the latest updates or reach out to [https://pgsharp.us](https://pgsharp.us) the maintainers for more details.
